### ISSUE TITLE
Review of the Aweome Vue list

### ISSUE THREAD

**Issue by brillout** [brillout/awesome-vue#6]
You are being pinged because you have at least one resource listed under the Awesome Vue list.

As per https://github.com/vuejs/awesome-vue/issues/836 we have reviewed the list and created a new one; [brillout/awesome-vue/README_CLEANED.md](https://github.com/brillout/awesome-vue/blob/master/README_CLEANED.md).

We removed many resources. Removal reasons can be found at [brillout/awesome-vue/README.md](https://github.com/brillout/awesome-vue/blob/master/README.md).

Projects using Vue.js have been reviewed by @Ffloriel and I reviewed all components & libraries.

I tried my best but I am sure that I did many mistakes. Let us know if you catch such mistake or if you disagree with a decision.

Also, many libraries are missing a note to state what Vue version is supported. Be sure that your library includes such note. Ideally right a the beginning of the readme text. (A badge or a GitHub tag can be easily overseen.)

I've not included libraries that have no English documentation as stated in https://github.com/brillout/awesome-vue/issues/5.

<br/>
<br/>

ping

@10quality @AStaroverov @Akryum @Alex-fun @AlexToudic @AlexandreBonaventure @Angarsk8 @Astray-git @Atinux @BirdEggegg @BosNaufal @Bubblings @CYBAI @CaiYiLiang @ChangJoo-Park @CodeFalling @Coffcer @CroudSupport @DanielZhu @David-Desmaisons @ElemeFE @F-loat @FE-Driver @FranckFreiburger @FranzSkuffka @GeoffZhu @Ginhing @Gomah @Grottolabs @GuillaumeLeclerc @Haixing-Hu @HerringtonDarkholme @JALBAA @Jack85 @JackGit @JosephusPaye @Justineo @Kashio @KoRiGaN @Kocisov @KyleRoss @LancerComet @MMF-FE @Ma63d @MacArthurJustin @MarkPieszak @MartyWallace @Mati365 @MatteoGabriele @MattyRad @Memkits @MetinSeylan @Metnew @MiCottOn @MillerRen @Monine @MopTym @MovingToHTTPS @N3-components @NeutroniumCore @NewDadaFE @NightCatSama @OYsun @OnsenUI @PeachScript @PeakTai @PetervMeijgaard @Plortinus @Polyconseil @QingWei-Li @Radyushin @RashadSaleh @RobinCK @Roeefl @SSENSE @ScreamZ @SevenOutman @SidKwok @SimulatedGREG @SkewedAspect @Socketize @Sopamo @SortableJS @TIGERB @TahaSh @Teddy-Zhu @Treri @Twiknight @Vanthink-UED @VueCluster @Vuedeux @Vuedo @Wanderxx @XeonPowder @XingzheFE @ZeroDark1991 @Zhangdroid @aarondfrancis @abdullah @airbnb @airyland @akifo @alidcastano @andylei18 @antongorodezkiy @apertureless @appcomponents @approach0 @arexio @arvidkahl @asika32764 @astralapp @atwxp @aweber1 @ayamflow @bbonnin @bkzl @blue0728 @bootstrap-vue @bpierre @bradstewart @brandonjpierce @brianvoe @bstavroulakis @budiadiono @buhrmi @c01nd01r @calebroseland @caolinjian @ceolter @cezardasilva @charliekassel @cheapsteak @chenz24 @cklmercer @cngu @codecasts @coderdiaz @compmeist @crossjs @cs1707 @cucygh @dalphyx @dangvanthanh @danielschmitz @danmademe @dargue3 @darrynten @davidmoshal @dealloc @declandewet @delay @devfake @dgerber @didierfranc @dkfbasel @doodlewind @drewjbartlett @dwightjack @ealves-pt @ecerroni @echoVic @eddyerburgh @eduardostuart @egoist @eliep @eljefedelrodeodeljefe @erguotou520 @ericmcdaniel @fancellu @fenivana @fergaldoyle @filipemeneses @firework @fishenal @forehalo @formly-js @foxbenjaminfox @freearhey @fritx @fundon @galenyuan @gismall @gluons @gocanto @greyby @groupwrite-io @gurghet @gustaYo @hal0gen @happilymarrieddad @happyDemon @haydenbbickerton @hchstera @heatherbooker @hedefalk @helpbase @hilongjw @hjeti @holic @huanleguang @iFgR @iatsiuk @icebob @ihanyang @ilyashubin @imcvampire @inca @irwansyahwii @italomaia @itsFrank @iview @jade-press @javisperez @jbaysolutions @jcc @jfelsinger @jiakeqi @jiangjiu @jinzhe @joearcher @joseluisq @jrainlau @julesbou @kaorun343 @karol-f @kartsims @katik @kazupon @kevinongko @kewah @kfei @kikyous @klummy @knpwrs @ktquez @ktsn @kylestlb @lajosbencz @lakb248 @laoshu133 @leaves4j @legeneek @leonardovilarinho @lian-yue @lithiumjake @liujians @lmk123 @locoslab @logaretm @lokyoung @loujiayu @lsycxyj @ludo237 @luojilab @madskristensen @mahmoudmy @marcosmoura @mark-hahn @matfish2 @matiastucci @matt-sanders @mbouclas @metricloop @mgyongyosi @mhayes @miaolz123 @mint- @mint-ui @misterGF @mlyknown @moip @monterail @moonou @mrgodhani @mtorromeo @mul14 @museui @mustardamus @myronliu347 @mysticatea @nblackburn @neposoft @nguyenvanduocit @nicolasbeauvais @niksmr @njleonzhang @nolimits4web @noru @nuxt @octref @okoala @oldj @pagekit @palerdot @pandao @partyka95 @paulpflug @pepf @petervmeijgaard @phanan @phoenixwong @phphe @pinguinjkeke @pluto1114 @pmsaue0 @postalservice14 @posva @pramjeet @processing @qazbnm456 @quasarframework @qxl1231 @rafaelpimpa @rahulgaba16 @rajabishek @rascada @ratiw @rawcreative @rayfranco @reinerBa @revue @rhyzx @ridermansb @rigor789 @robinvdvleuten @rodzzlessa24 @rokups @ropbla9 @rpkilby @rstoenescu @sabrinaluo @sagalbot @saivarunk @saman @samcrosoft @samiheikki @samturrell @sandeepk01 @sant123 @santiblanko @sapjax @scaccogatto @scottbedard @scriptPilot @se-panfilov @sebastian-software @sejr @semisleep @shakee93 @shershen08 @shhdgit @shinygang @shiye515 @simplesmiler @sinchang @sjerrys @skyrpex @staskjs @steven5538 @storyblok @suguangwen @sunabozu @super-dev @superlloyd @superman66 @superwf @surfbirdapp @surmon-china @surveyjs @syropian @taoeffect @termosa @text-mask @the6thm0nth @thelinuxlich @theomessin @thetutlage @tyllo @typicode @uniquezhuo @vace @varHarrie @viko16 @villeristi @vmaimone @vue-bulma @vue-comps @vue-play @vue-typed @vuedev-com @vuejs-br @vuetifyjs @vuets @vuikit @wangdahoo @waynecz @weavingbird @websemantics @weibangtuo @weifeiyue @weilao @weinot @weizhenye @wlada @wovue @wyzant-dev @xanf @xiaokaike @xiaoluoboding @xlsdg @xpepermint @xrado @yang-wei @ycwalker @ye-will @yetaai @yjj5855 @yodfz @yujiahaol68 @zerostatic @zhangxin840 @zhouzhuojie @zllovesuki @znck @zurb @zxdong262

**Comment by brillout** [comment]
ping

@MattyRad @Memkits @MetinSeylan @Metnew @MiCottOn @MillerRen @Monine @MopTym @MovingToHTTPS @N3-components @NeutroniumCore @NewDadaFE @NightCatSama @OYsun @OnsenUI @PeachScript @PeakTai @PetervMeijgaard @Plortinus @Polyconseil @QingWei-Li @Radyushin @RashadSaleh @RobinCK @Roeefl @SSENSE @ScreamZ @SevenOutman @SidKwok @SimulatedGREG @SkewedAspect @Socketize @Sopamo @SortableJS @TIGERB @TahaSh @Teddy-Zhu @Treri @Twiknight @Vanthink-UED @VueCluster @Vuedeux @Vuedo @Wanderxx @XeonPowder @XingzheFE @ZeroDark1991 @Zhangdroid @aarondfrancis

**Comment by brillout** [comment]
ping

@abdullah @airbnb @airyland @akifo @alidcastano @andylei18 @antongorodezkiy @apertureless @appcomponents @approach0 @arexio @arvidkahl @asika32764 @astralapp @atwxp @aweber1 @ayamflow @bbonnin @bkzl @blue0728 @bootstrap-vue @bpierre @bradstewart @brandonjpierce @brianvoe @bstavroulakis @budiadiono @buhrmi @c01nd01r @calebroseland @caolinjian @ceolter @cezardasilva @charliekassel @cheapsteak @chenz24 @cklmercer @cngu @codecasts @coderdiaz @compmeist @crossjs @cs1707 @cucygh @dalphyx @dangvanthanh @danielschmitz @danmademe @dargue3

**Comment by brillout** [comment]
ping

@darrynten @davidmoshal @dealloc @declandewet @delay @devfake @dgerber @didierfranc @dkfbasel @doodlewind @drewjbartlett @dwightjack @ealves-pt @ecerroni @echoVic @eddyerburgh @eduardostuart @egoist @eliep @eljefedelrodeodeljefe @erguotou520 @ericmcdaniel @fancellu @fenivana @fergaldoyle @filipemeneses @firework @fishenal @forehalo @formly-js @foxbenjaminfox @freearhey @fritx @fundon @galenyuan @gismall @gluons @gocanto @greyby @groupwrite-io @gurghet @gustaYo @hal0gen @happilymarrieddad @happyDemon @haydenbbickerton @hchstera @heatherbooker @hedefalk

**Comment by brillout** [comment]
ping

@helpbase @hilongjw @hjeti @holic @huanleguang @iFgR @iatsiuk @icebob @ihanyang @ilyashubin @imcvampire @inca @irwansyahwii @italomaia @itsFrank @iview @jade-press @javisperez @jbaysolutions @jcc @jfelsinger @jiakeqi @jiangjiu @jinzhe @joearcher @joseluisq @jrainlau @julesbou @kaorun343 @karol-f @kartsims @katik @kazupon @kevinongko @kewah @kfei @kikyous @klummy @knpwrs @ktquez @ktsn @kylestlb @lajosbencz @lakb248 @laoshu133 @leaves4j @legeneek @leonardovilarinho @lian-yue

**Comment by brillout** [comment]
ping

@lithiumjake @liujians @lmk123 @locoslab @logaretm @lokyoung @loujiayu @lsycxyj @ludo237 @luojilab @madskristensen @mahmoudmy @marcosmoura @mark-hahn @matfish2 @matiastucci @matt-sanders @mbouclas @metricloop @mgyongyosi @mhayes @miaolz123 @mint- @mint-ui @misterGF @mlyknown @moip @monterail @moonou @mrgodhani @mtorromeo @mul14 @museui @mustardamus @myronliu347 @mysticatea @nblackburn @neposoft @nguyenvanduocit @nicolasbeauvais @niksmr @njleonzhang @nolimits4web @noru @nuxt @octref @okoala @oldj @pagekit

**Comment by brillout** [comment]
ping

@palerdot @pandao @partyka95 @paulpflug @pepf @petervmeijgaard @phanan @phoenixwong @phphe @pinguinjkeke @pluto1114 @pmsaue0 @postalservice14 @posva @pramjeet @processing @qazbnm456 @quasarframework @qxl1231 @rafaelpimpa @rahulgaba16 @rajabishek @rascada @ratiw @rawcreative @rayfranco @reinerBa @revue @rhyzx @ridermansb @rigor789 @robinvdvleuten @rodzzlessa24 @rokups @ropbla9 @rpkilby @rstoenescu @sabrinaluo @sagalbot @saivarunk @saman @samcrosoft @samiheikki @samturrell @sandeepk01 @sant123 @santiblanko @sapjax @scaccogatto

**Comment by brillout** [comment]
ping

@scottbedard @scriptPilot @se-panfilov @sebastian-software @sejr @semisleep @shakee93 @shershen08 @shhdgit @shinygang @shiye515 @simplesmiler @sinchang @sjerrys @skyrpex @staskjs @steven5538 @storyblok @suguangwen @sunabozu @super-dev @superlloyd @superman66 @superwf @surfbirdapp @surmon-china @surveyjs @syropian @taoeffect @termosa @text-mask @the6thm0nth @thelinuxlich @theomessin @thetutlage @tyllo @typicode @uniquezhuo @vace @varHarrie @viko16 @villeristi @vmaimone @vue-bulma @vue-comps @vue-play @vue-typed @vuedev-com @vuejs-br

**Comment by brillout** [comment]
ping

@vuetifyjs @vuets @vuikit @wangdahoo @waynecz @weavingbird @websemantics @weibangtuo @weifeiyue @weilao @weinot @weizhenye @wlada @wovue @wyzant-dev @xanf @xiaokaike @xiaoluoboding @xlsdg @xpepermint @xrado @yang-wei @ycwalker @ye-will @yetaai @yjj5855 @yodfz @yujiahaol68 @zerostatic @zhangxin840 @zhouzhuojie @zllovesuki @znck @zurb @zxdong262

**Comment by dealloc** [comment]
How should updates be communicated?

**Comment by danmademe** [comment]
I like the cleanup. I'll also make a note of what version express-vue uses. But it's always latest as I use greenkeeper.io to keep all dependencies latest and tested.

**Comment by brillout** [comment]
@dealloc if you are confident about your update -> PR, otherwise let's discuss your update here

**Comment by javisperez** [comment]
Good, question:

I see that there's no Modals in the list and just today i've added a modal component. I've sent a PR today, but dont if modal windows arent allowed anymore or something. I've search in  brillout/awesome-vue/README.md but couldnt find any related info.

**Comment by dealloc** [comment]
@brillout I have explained advantage of an IoC container in the introduction section, I'm not quite sure what more information you'd like to see. I'd be more than willing to add whatever you feel is missing

**Comment by danmademe** [comment]
If I work on a vuejs version number badge would that be of interest?

**Comment by scottbedard** [comment]
@brillout I've addressed your concern about stating which version of Vue my project uses.
https://github.com/scottbedard/oc-vuetober-theme

**Comment by brillout** [comment]
@javisperez 

> I've sent a PR today

Can't find it, where is it?

@dealloc 

I remember vuec. I liked the introduction explaining DI. But it doesn't develop in what use cases DI is actually helpful. In short; What was your motivation to write vuex other than implementing DI? E.g. testing? I'll be happy to add it to the new list.

@danmademe 

Can you create an issue for that? We could discuss it there.

**Comment by javisperez** [comment]
@brillout  it's on the original repo:
https://github.com/vuejs/awesome-vue/pull/934

But I suppose I have to make it to this new clean version, I'll make the PR tonight

**Comment by brillout** [comment]
> on the original repo

Ah, I only checked the merged ones.

> But I suppose I have to make it to this new clean version, I'll make the PR tonight

Yea that would be awesome. I'll review it then.

**Comment by MartyWallace** [comment]
So is there something you need me to do for my keyboard component to be included in the cleaned list or is this just an FYI that it doesn't belong there altogether?

**Comment by fergaldoyle** [comment]
vue-form got updated to work with 2.0 a couple of days ago.

**Comment by brillout** [comment]
@MartyWallace The ticket is precisely about discussing this type of situations where lib authors disagree with a decision I took. I didn't add your lib because I didn't find a live demo, otherwise I would have added it. If you add a live demo, I'll happily add it to the new list.

@fergaldoyle neat, I'll add it to the new list

**Comment by filipemeneses** [comment]
@brillout vue-chalkboard isn't just a demo, can be used as component in other applications and there is a [doc](https://github.com/filipemeneses/vue-chalkboard/blob/master/docs/README.md).

**Comment by MartyWallace** [comment]
@brillout Ahh I see, no problem at all. I can sort that out within a couple hours.

**Comment by dealloc** [comment]
@brillout I updated the documentation of vuec to be a bit more thorough, I'd like your opinion if you have some spare time.

You can find the updated documentation [here](https://github.com/dealloc/vuec/tree/dealloc/docs#why-use-dependency-injection)

**Comment by brillout** [comment]
@filipemeneses I've seen the doc but I still hesitated on that back then. I just didn't find many use cases why someone would want to add a chalkboard to their app. What I liked about @MartyWallace 's virtual keyboard doc was `I initially developed this component for the KitKat Chocolatory kiosk interfaces that we set up in Melbourne`. That made me understand why someone would want that.

@MartyWallace Awesome

@dealloc Love it, perfect, I'll add to the new list

**Comment by brillout** [comment]
@fergaldoyle @dealloc Actually, it would be great if guys can PR your libs into README_CLEANED.md. Thanks! (The format is ` - [npm-package-name](https://github.com/org/repo) - Description of the lib.`.)

**Comment by dealloc** [comment]
@brillout no problem, under what category should I place vuec? I don't see any category it fits in currently

**Comment by javisperez** [comment]
@brillout @dealloc question is a good question :D what category should I use for a Modal/Dialog (like confirm, alert) component? UI Layout?

**Comment by filipemeneses** [comment]
@brillout I made because I needed in my app.

**Comment by brillout** [comment]
@dealloc I'd actually put it in two categories; `Dev Tools > Test` and `Utilities > Miscellaneous`. (FYI your lib is the second one that is in >1 category.)

@javisperez :D Feel free to create a new category in `UI Components`. (Not in `UI Layout`, see description `Layout for the overall / main view` of `UI Layout`.)

**Comment by villeristi** [comment]
I'd ditch the hardcoded stars & add some badges instead. One reason less for somewhat redundand PR's.

**Comment by MartyWallace** [comment]
@brillout I've updated my package, hopefully it seems more suitable now:

* Added a [demo](http://vue-keyboard.martywallace.com/).
* Updated the component to [use NPM](https://www.npmjs.com/package/vue-keyboard) instead of Bower.
* Converted it to [a single file `.vue` component](https://github.com/MartyWallace/vue-keyboard/blob/master/src/keyboard.vue) as that appears to be the standard.
* Noted the Vue version requirement in the README.

Let me know if you think anything else should be done.

**Comment by Treri** [comment]
@brillout 
vue-ios-alertview was just born with Vue 2. So you can safely add it to the new cleaned list.

P.S: There's no __General Plugins/Directives__ group in cleaned list, and I don't know which category vue-ios-alertview should belong to, so I didn't make a PR.

**Comment by crossjs** [comment]
@brillout 
I've updated the PLATO boilerplate, which README is entirely in english now.
Thanks.

**Comment by matt-sanders** [comment]
@brillout I'm wondering if there's a better section for vue-formly. It's not really a validation library although it does handle some validation internally. Maybe just directly under Forms would be a better place for it?

**Comment by njleonzhang** [comment]
@brillout 
[vue-data-tables](https://github.com/njleonzhang/vue-data-tables) from a lib KB size POV, the dependency on element-ui is quite a bad thing

If user already use element-ui, then this lib can be seamless used. If some one don't want to import whole element-ui, then they can just import some components of element-ui,  which are dependent of `vue-data-tables`. I can describe this in Readme, does this make sense to you?

**Comment by gluons** [comment]
> added value too small

@brillout What is "added value too small"? Could you give me more information?

**Comment by MarkPieszak** [comment]
@brillout my aspnet vue starter has the version in it, was it removed or something?

**Comment by cngu** [comment]
What is "too specific" about vue-typer? I tried to add as many configurable options as possible. 

If the problem it solves is too specific, I'd argue that's fine for vue-typer because it's a specific (but configurable and stylable) UI element, and the library shouldn't try to do too much more than advertised.

**Comment by noru** [comment]
@brillout Hi Thanks fro the notice. I'm the owner of vue-resource-mock which was marked:
`vue-resource-mock doesn't state v2 support (otherwise good)`

The thing is, this component, just as the vue-resource, require no specific version of Vue. As long as the vue-resource doesn't push breaking changes, it should work.

I don't think I could just add "Vue/Vue2 compatible" because vue-resource doesn't state that in their README. I would like to add something like "Rely on vue-resource and no specific version of Vue is required", it that ok?

**Comment by leonardovilarinho** [comment]
@brillout Could send me because my plugins have been removed?

https://github.com/leonardovilarinho/vue-multilanguage

https://github.com/leonardovilarinho/vue-acl

**Comment by pluto1114** [comment]
@brillout I've updated the vue-music163, which all pages is entirely in english now.
Thanks.

**Comment by aweber1** [comment]
@brillout Hi, it looks like many of the 'Examples' items were removed from the new clean version. Mine is https://github.com/aweber1/tour-of-heroes-vue2 

I completely understand if it's no longer relevant in the new clean list, but as there was no removal reason next to it, just wondering if there's something I need to do in order to keep the project on the list.

Thanks!

**Comment by kevinongko** [comment]
Hi @brillout & @Ffloriel , I've updated [vue-instagram](https://github.com/kevinongko/vue-instagram) and [vue-numeric](https://github.com/kevinongko/vue-numeric) adding vue version and better docs. Thanks.

**Comment by italomaia** [comment]
Hello. Could I get a description of why vuejs-python was removed? I would like to work on the addressed problems.

**Comment by varHarrie** [comment]
@brillout I am the owner of `vuex-action`, I 've update Readme.md and add examples.

**Comment by erguotou520** [comment]
Hey, guys. I have pulled a repo to provide vue badge in your readme file, see https://github.com/erguotou520/vue-version-badge
BTW, I am working on updating `vue-fullstack` project. Thank you for your hard work. @brillout

**Comment by CaiYiLiang** [comment]
@aweber1 the question you point out is also what I want to ask :smile: 
@brillout I wonder whether the 'Example' will contibue to be on list , thought the items under  'Example' list is not a e2e big project sometimes , but I think the basic examples can help some newer or thoese who want to experience vue by writing demos . My demo is as below : https://github.com/CaiYiLiang/simply-calculator-vuejs   , https://github.com/CaiYiLiang/vue-demos/tree/master/wikipediaViewer-vuejs , https://github.com/CaiYiLiang/vue-demos/tree/master/shoppingcart-vuejs   

Thanks for your effort at advanced.

**Comment by pinguinjkeke** [comment]
Thank you.
Updated my package's README.md with note that says it supports both Vue and Vue 2

**Comment by lsycxyj** [comment]
Thanks a lot, I don't have anything need to be updated though.

**Comment by TIGERB** [comment]
@brillout [easy-vue](https://github.com/TIGERB/easy-vue) state it support vue2

**Comment by joseluisq** [comment]
@brillout 
> [vue-vpaginator](https://github.com/joseluisq/vue-vpaginator) seems too much to include jQuery and boostrap for a pagination component

Yeah, **maybe** (in contrast, developers have many alternatives here too). So [vue-vpaginator](https://github.com/joseluisq/vue-vpaginator) requires jQuery but doesn't need the Bootstrap script, it uses the Bootstrap pagination markup only (plain HTML).

Anyway, I will go to specify it and add the jQuery dependency.
Thanks!

**Comment by ludo237** [comment]
My [vuejs-carousel](https://github.com/ludo237/vuejs-carousel) has full support for V2, it does not have heavy dependency, aside from moment.js and it's different from the other carousels in the new list because it includes a theater mode plus comment sections inspired by Facebook

**Comment by happyDemon** [comment]
I have added a Vue2 topic, does that suffice?
https://github.com/happyDemon/vue-echo

**Comment by weizhenye** [comment]
@brillout vue-highcharts supports both v1 and v2.

**Comment by dangvanthanh** [comment]
@brillout [vue-ckeditor2](https://github.com/dangvanthanh/vue-ckeditor2) supports both v1 and v2 too.

**Comment by superwf** [comment]
@brillout what "is not a library" means?  what should I fix?

**Comment by RashadSaleh** [comment]
@brillout I have included the statement: "Tested to work with Vue2." near the top of README.md for [vuex-basement](https://github.com/RashadSaleh/vuex-basement).

**Comment by mrgodhani** [comment]
@brillout  I already have that Vue2.0 is on it's way.

**Comment by xanf** [comment]
hi @brillout I really appreciate your effort. Just a note - [vuex-redux-saga](https://github.com/xanf/vuex-redux-saga) does not a simple replacement of "actions" provided in vuex - it is used for handling complex scenarios when order of mutations could be arbitrary, also when you need things running in parallel, etc.  A good explanation by my colleague on real life project (with React though) - https://goshakkk.name/lazy-auth-redux-saga-flow/ 

Let me know If I'm missing the point how i can build easily such "complex" functionalities with pure vuex

**Comment by rpkilby** [comment]
> We removed many resources. Removal reasons can be found at brillout/awesome-vue/README.md.

@brillout - could you add a note near this line that tells authors/maintainers what to do once the original issue has been resolved?

**Comment by lokyoung** [comment]
Hello @brillout . So much thanks for your work.
I'm the author of [vuejs-paginate](https://github.com/lokyoung/vuejs-paginate). I have stated that the component is working for Vue 2. Also add a PR. You can check it. Thanks!

**Comment by brillout** [comment]
@MartyWallace Great! Make a PR and I'll merge it.

@Treri Feel free to create a new category; How about `Overlay` under `UI Components`? Please coordinate with @javisperez and his PR https://github.com/brillout/awesome-vue/pull/12

@crossjs 👍 for the translation. But the demo http://plato.crossjs.com/#/ doesn't seem to work? I can't make any action on Chrome for desktop.

@matt-sanders true, I've improved the categorization, better now?

@njleonzhang yes ok. Also would be nice if everything in https://njleonzhang.github.io/vue-data-tables/#/en and in the readme is in English.

**Comment by sant123** [comment]
@brillout now [vuejs-uib-pagination](https://github.com/sant123/vuejs-uib-pagination) specifies the support for Vue.js 2.0 only.

**Comment by rpkilby** [comment]
@brillout vue-super has been updated with tests and explicit Vue 2.x compatibility.

**Comment by reinerBa** [comment]
I have created a npm package for my plugin

> [vue-responsive](https://reinerba.github.io/Vue-Responsive/) no npm package (otherwise good)
and i will update the readme for my other plugin in the future :)

**Comment by Ffloriel** [comment]
@pluto1114 Good. Could you update also the screenshots of the application?

**Comment by brillout** [comment]
@gluons sure, my perspective; What is the complexity implemented by your library? Implementing a wrapper around highlight.js is easy. If I would need one then I would just implement it myself without even searching for a library. Since it's only couples of LOC.

@MarkPieszak @aweber1 @CaiYiLiang @TIGERB We've mistakenly skipped the examples part. Please make PRs!

@cngu I was mentioning the problem it solves. I still included it in the new list though.

@noru yes, please make a PR afterwards.

@leonardovilarinho please check the README.md in this repo.

@kevinongko would be great to have a live demo for vue-numeric, seems good otherwise. As for vue-instagram, I'd say that it's too specific and the added value too small.

@italomaia where on the awesome list is it? Can't find any `vuejs-python` resource.

@varHarrie make a PR:)

@pinguinjkeke @lsycxyj thanks:)

@joseluisq Actually, I'm confused as to why you need jQuery for it? I'd personally rather use https://github.com/lokyoung/vuejs-paginate since it doesn't depend on jQuery

@ludo237 I'd decouple the carousel part from the comment part

@happyDemon I personally keep overlooking these GitHub tags

@weizhenye Great, please make a PR

@dangvanthanh is it on the Awesome Vue list? Can't find it

@superwf I meant that your library's use case seems too specific

@RashadSaleh great

@mrgodhani cool

@rpkilby done

@lokyoung it's merged

@sant123 nice

@rpkilby 👍 make a PR and I'll merge it

@reinerBa make a PR that I will merge

**Comment by brillout** [comment]
@MarkPieszak @aweber1 @CaiYiLiang @TIGERB Examples are now part of the new list.

**Comment by matt-sanders** [comment]
@brillout perfect, thanks!

**Comment by brillout** [comment]
@xanf how about adding a short use case that is easy to implement with vuex-redux-saga but difficult with vuex actions & mutations to the readme?

**Comment by kaorun343** [comment]
@brillout 
Hi.

I understand that your work is so important to Vue's community, but **you should have chosen the other way** How many emails did I receive from this issue?????? Those who not interested in the new list, not interested in updating their library or readme also receives them.

Are you going to continue all discussions at here? Please, please, lock this issue and ask them to make issue per user.

**Comment by brillout** [comment]
@kaorun343 

Sorry and you are right, a new issue per discussion is better.

(Note that you can click on the "Unsubscribe" button on the right to stop receiving emails.)

**Comment by brillout** [comment]
To everyone: please create a new issue per discussion.
